scripts/comparison/parse_perf_results.py

```diff
@@ -0,0 +1,150 @@
+import sys
+import getopt
+import os
+import glob
+from html_report import HtmlReportSection, HtmlReport
+from perf_report import NTTTCPLogsReader
+
+
+def get_arameters(argv):
```

Comment:
general comments that apply to each python script:
1. read pep8
2. use pep8/flake8
3. variable_name and not variableName
4. method_name and not anything else
5.ClassName